test/paths.c

```diff
@@ -85,6 +85,10 @@ int main(void)
     TEST_NORMALIZE("/foo", "/foo");
 #endif
 
+#if defined(_WIN32) && (!defined(HAVE_PATHCCH) || !HAVE_PATHCCH)
```

Comment:
isn't `HAVE_PATHCCH` always defined? by our meson code I mean

Reply:
It is checked inside `features['win32-desktop']` condition. Technically I could move it out if you prefer. What do you think?

https://github.com/mpv-player/mpv/blob/06ec0319dbd3edb44bf383a292bc412ce9a5d021/meson.build#L500-L514

Reply:
It's fine, was just wondering.